index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="./Components/Pages/Styles/main.css">
+  <title>Game Ranking: Home page</title>
+</head>
+
+<body>
```

Comment:
The Github Pages link is now working but there are a few issues with it. As you can see there are quite a few files that are not being found properly and are affecting your app. Feature speakers nor the program sponsors are getting created due to the lack of your JS file 😥

![image](https://user-images.githubusercontent.com/15807343/157766774-242251f6-e1a4-4d8f-95ae-a19d0c4021fa.png)

I downloaded your project and its looking fine on the live server 🔥 So it must be a path issue